code/game/machinery/telecomms/machine_interactions.dm

```diff
@@ -255,3 +255,16 @@
 	if(issilicon(user) || in_range(user, src))
 		return TRUE
 	return FALSE
+
+/obj/machinery/telecomms/multitool_act_secondary(mob/living/user, obj/item/tool)
+	var/obj/item/multitool/multitool = tool
+	var/obj/machinery/telecomms/telecomms_machine = multitool.buffer
+	if(QDELETED(multitool.buffer)||!multitool.buffer || !istype(telecomms_machine) || telecomms_machine == src) //sanity + user sanity
+		return FALSE
+	if(multitool.buffer in links)
+		to_chat(user, span_notice("Unlinked [telecomms_machine] ([telecomms_machine.id]) from the [src]."))
+		remove_link(multitool.buffer,user)
+	else
+		to_chat(user, span_notice("Linked [telecomms_machine] ([telecomms_machine.id]) to the [src]."))
```

Comment:
These should be balloon messages

Reply:
too long for balloon messages

Reply:
> too long for balloon messages

indeed

Reply:
which is why you make the balloon message `linked` and `unlinked`

Reply:
it literally has the name of the machine and the id of the machine so you can look at chat log and see what you linked or unlinked to what incase you inevitably fuck up

Reply:
That's fair; I just know we try to minimize to_chat calls for stuff that doesn't need to be logged(cn)

Reply:
if i understand that balloon alerts are supposed to be short then trying to cram in machine name and ID would be too long and even then you cant just look at chat and go "aw nuts i linked the hub to the processor/clown/whatever" with balloon alerts